internal/v2/application/command.go

```diff
@@ -236,7 +238,7 @@ func (c *CommandProcessor) ReadCommand() (res dtos.Event, e edgexErr.EdgeX) {
 
 func (c *CommandProcessor) WriteDeviceResource() edgexErr.EdgeX {
 	lc := bootstrapContainer.LoggingClientFrom(c.dic.Get)
-	lc.Debugf("Application - writeDeviceResource: writing deviceResource: %s", c.deviceResource.Name, common.CorrelationHeader, c.correlationID)
+	lc.Debug(fmt.Sprintf("Application - writeDeviceResource: writing deviceResource: %s", c.deviceResource.Name), common.CorrelationHeader, c.correlationID)
```

Comment:
use Debugf

Reply:
decided to add the correlationID into log message in application layer.